src/entity/transactions/sub-transaction-row.ts

```diff
@@ -41,7 +41,7 @@ export default class SubTransactionRow extends BaseEntity {
   })
   public amount: number;
 
-  @ManyToOne(() => Invoice, { nullable: true })
+  @ManyToOne(() => Invoice, { nullable: true, onDelete: 'CASCADE' })
```

Comment:
Same here... If the invoice is deleted, this subtransactionrow will be deleted too? I would say you would use `RESTRICT`, because you do not want to delete a subtransactionrow that you have invoiced